Gruntfile.js

```diff
@@ -76,7 +76,10 @@ module.exports = function (grunt) {
         stylus: {
             minerva: {
                 files: [{
-                    src: ['<%= plugin.minerva.external %>/stylesheets/main.styl'],
+                    src: [
+                        '<%= plugin.minerva.external %>/stylesheets/main.styl',
+                        '<%= plugin.minerva.root %>/node_modules/colorbrewer/colorbrewer.css'
```

Comment:
No, I'm doing something like that for bringing in datatables.  I'll move these there on that PR.